src/main/java/eu/europeana/metis/sandbox/repository/TransformXsltRepository.java

```diff
@@ -1,8 +1,9 @@
 package eu.europeana.metis.sandbox.repository;
 
 import eu.europeana.metis.sandbox.entity.TransformXsltEntity;
+import java.util.Optional;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface TransformXsltRepository extends JpaRepository<TransformXsltEntity, Integer> {
-
+  Optional<TransformXsltEntity> findFirstByIdIsNotNullOrderByIdAsc();
```

Comment:
why ```findFirstByIdIsNotNullOrderByIdAsc```? it is a primary key so never should be ```null``` the value is ```SERIAL``` according to the DB definition. Can you validate that only by ```findFirstByIdOrderByIdAsc``` does the same work? if it does it please replace it.

Reply:
This one I maintained the original JPA query. 
Since the `findFirstByIdOrderByIdAsc` requires an `id` as parameter. 
I wish to get the first that isn't `null` so it works as expected, without a parameter